<launch>
  <node name="data_collector" pkg="data_collector" type="data_collector" output="screen">
    <remap from="~input_point_cloud" to="/realsense2/depth/color/points" />
    <remap from="~input_color_image" to="/realsense2/color/image_raw" />
    <remap from="~input_depth_image" to="/realsense2/aligned_depth_to_color/image_raw" />
    <remap from="~input_camera_information" to="/realsense2/aligned_depth_to_color/camera_info" />
    <remap from="~reset" to="/data_collector_node/reset" />
    <remap from="~capture_data" to="/data_collector_node/capture_point_cloud" />
    <remap from="~data_collected" to="/data_collected" />
    <param name="fusion_frame" value="base"/>
    <rosparam param="bounding_box">[0.28,0.65,-0.44,0.39,0.01,0.50]</rosparam>
  </node>
</launch>
